spec.md

```diff
@@ -179,6 +218,49 @@ physically present in the final circuit. Refer to the description of the
 instance statement for details on how to instantiate a module
 ([@sec:instances]).
 
+### Public Modules
+
+A public module is a module that is marked with the `public`{.firrtl} keyword.
+A public module has a defined ABI---knowing the FIRRTL representation is
+sufficient to know its compiled representation.  This means that a public module
+is available external to this circuit.
```

Comment:
I don't think this strictly follows.  I could know the representation, but migrate all the contents to the in-circuit instantiators.  Public requires stronger semantics.

Reply:
This is in fact stated later, which makes this partially redundant, but in a way which emphasizes just a part as though that was the most important thing.

Reply:
I'll drop this and switch to simpler language about public being capable of being instantiated externally.